packages/react-google-maps-api/src/components/directions/DirectionsRenderer.md

```diff
@@ -1,262 +1,323 @@
+          <div className="form-group custom-control custom-radio mr-4">
+            <input
+              id="WALKING"
+              className="custom-control-input"
+              name="travelMode"
+              type="radio"
+              checked={
+                directionsFormValue.travelMode ===
+                google.maps.TravelMode.WALKING
+              }
+              onChange={checkWalking}
+            />
+            <label className="custom-control-label" htmlFor="WALKING">
+              Walking
+            </label>
+          </div>
+        </div>
+        <button className="btn btn-primary" type="button" onClick={onClick}>
+          Build Route
+        </button>
+      </div>
 
-        <div className='map-container'>
-          <GoogleMap
-            // required
-            id='direction-example'
-            // required
-            mapContainerStyle={{
-              height: '400px',
-              width: '100%'
-            }}
-            // required
-            zoom={2}
-            // required
-            center={{
-              lat: 0,
-              lng: -180
-            }}
-            // optional
-            onClick={this.onMapClick}
-            // optional
-            onLoad={map => {
-              console.log('DirectionsRenderer onLoad map: ', map)
-            }}
-            // optional
-            onUnmount={map => {
-              console.log('DirectionsRenderer onUnmount map: ', map)
-            }}
-          >
-            {
-              (
-                this.state.destination !== '' &&
-                this.state.origin !== ''
-              ) && (
-                <DirectionsService
-                  // required
-                  options={{ // eslint-disable-line react-perf/jsx-no-new-object-as-prop
-                    destination: this.state.destination,
-                    origin: this.state.origin,
-                    travelMode: this.state.travelMode
-                  }}
-                  // required
-                  callback={this.directionsCallback}
-                  // optional
-                  onLoad={directionsService => {
-                    console.log('DirectionsService onLoad directionsService: ', directionsService)
-                  }}
-                  // optional
-                  onUnmount={directionsService => {
-                    console.log('DirectionsService onUnmount directionsService: ', directionsService)
-                  }}
-                />
-              )
-            }
+      <div className="map-container">
+        <GoogleMap
+          // required
+          id="direction-example"
+          // required
+          mapContainerStyle={containerStyle}
+          // required
+          zoom={2}
+          // required
+          center={center}
+          // optional
+          onClick={onMapClick}
+          // optional
+          onLoad={(map) => {
+            console.log('DirectionsRenderer onLoad map: ', map);
+          }}
```

Comment:
here too!

Reply:
should i remove them or transfer them o a callback function instead?